### PR TITLE
Python 3 OpenGEE Server

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Migration of OpenGEE Server to python 3.

To test (should be done on both CentOS and Ubuntu):
1. Build and install according to the build docs (https://github.com/tst-cjeffries/earthenterprise/blob/python_3_server/earth_enterprise/BUILD_RHEL_CentOS.md)
2. Run the tutorials.
`sudo /opt/google/share/tutorials/fusion/download_tutorial.sh`
`sudo cp earthenterprise/earth_enterprise/src/fusion/tools/gee_test/tutorial_files/providers.xml /gevol/assets/.userdata/`
`sudo env "PATH=$PATH:/opt/google/bin" earthenterprise/earth_enterprise/src/fusion/tools/gee_test/run_tutorial.sh`
`sudo env "PATH=$PATH:/opt/google/bin" earthenterprise/earth_enterprise/src/fusion/tools/gee_test/run_historical.sh`
Then wait until the tutorials are finished building. This can be monitored in with the status scripts in that same directory.
`sudo env "PATH=$PATH:/opt/google/bin" earthenterprise/earth_enterprise/src/fusion/tools/gee_test/publish_tutorial.sh`
`sudo env "PATH=$PATH:/opt/google/bin" earthenterprise/earth_enterprise/src/fusion/tools/gee_test/publish_historical.sh`
3. Verify the tutorials. This is just a matter of opening them in EC (browser for 2d) and inspecting the imagery, vector, and terrain. On historical databases, historical imagery should work.
4. Note that the various searches, snippets, cutter, and wms don't work, so they do not need to be tested.
5. Open the Diagnostics page on the OpenGEE Admin page and verify Python 3 is being used. Version number tests are expected to fail due to branch tagging issues.
6. Verify that databases can be published/unpublished on the OpenGEE Admin page.
7. Probably should test virtual hosts (https://t-sciences.jira.com/wiki/spaces/GEE/pages/416318328/Virtual+Host+Tests)
8. Admin console tests also (https://t-sciences.jira.com/wiki/spaces/GEE/pages/416153837/Admin+Console+Tests)
9. Verify the uninstall works.

I may have missed a thing or two, so If anything else you feel should be tested, feel free.

### CODE REVIEW

File: scripts/install_python.sh [hunk 1]
```diff
@@ -102,8 +102,9 @@ if [ $DO_INSTALL == "yes" ]; then
     mkdir $TMP_27
     tar -xzf $SOURCE_27 -C $TMP_WORKSPACE
     cd $TMP_27
-    ./configure --prefix="$INSTALL_PREFIX"
+    ./configure --enable-shared --prefix="$INSTALL_PREFIX"
     make && make altinstall
+    sudo ldconfig
     sudo python2.7 -m ensurepip
```
Comment (tst-ccamp) [inline scripts/install_python.sh:108]: I'm testing on Ubuntu and it seems that pip wasn't installed. Strangely, I'd assume that this line should have made sure it's installed? I had to use `sudo apt install python-pip`. I guess this is only run if python 2.7 isn't already installed. Probably easiest to just add python-pip to the list of python 2.7 packages to install in the Build Readme

**Comment by tst-ccamp** [comment]
Finished testing and everything worked except for my one comment about pip

File: earth_enterprise/src/third_party/SConscript [hunk 2]
```diff
@@ -65,9 +65,9 @@ if third_party_env['bundle_3rd_libs']:
       'pyyaml',
```
Comment (tst-ccamp) [inline earth_enterprise/src/third_party/SConscript:65]: Minor: needs to have the header updated for Copyright date

**Review by tst-ccamp — APPROVED** [review]
Just a couple minor issues, otherwise LGTM!

**Review by tst-dsugar — APPROVED** [review]
This is going to the migration branch....

**Pull request merged**